src/aibs_informatics_core/models/demand_execution/metadata.py

```diff
@@ -29,3 +32,62 @@ class DemandExecutionMetadata(SchemaModel):
         mm_field=ListField(StringField(), allow_none=True),
         default=None,
     )
+
+    @property
+    def tag(self) -> Optional[str]:
+        """Return all tags as a comma separated string if available.
+
+        Example:
+            >>> metadata = DemandExecutionMetadata(tags={"key": "value"})
+            >>> metadata.tag
+            'key=value'
+            >>> metadata = DemandExecutionMetadata(tags={"key": "key"})
+            >>> metadata.tag
+            'key'
```

Comment:
What would be the downsides of only supporting "key": "value" tag format?

i.e.
```
           >>> metadata = DemandExecutionMetadata(tags={"key": "key"})
            >>> metadata.tag
            'key=key'
```

Seems like that would simplify a lot? Is this only so we can maintain backwards compatibility? If so, I'd be more in favor of just dumping the backwards compatibility and doing  a datafix.